tests/cases/fourslash/completionsInExport.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts" />
+
+////const a = "a";
+////type T = number;
+////export { /**/ };
+
+verify.completions({
+  marker: "",
+  exact: ["a", "T"]
+});
+
+edit.insert("a");
+
+verify.completions({
+  marker: "",
+  exact: ["T"]
+});
```

Comment:
Can you also add tests for ` export  { a,  /*here*/}`, `export { a as /*here*/}`, `export { a as b, /*here*/ }`

Reply:
This brings up a good point: I was filtering out symbols that were already exported, but it’s valid to export the same thing several times if you rename:

```ts
export { T, T as T2, T as T3, T as T4, ... }
```

Do you think I should eliminate the filtering?

Reply:
I think instead of filtering you should sort text of those at lower priority ?

Reply:
I don't know how practical it is, but another option might be to include the `as` the second time it's committed.

Reply:
@sheetalkamat how would you set the sort text here? It defaults to `LocationPriority`, and the other enum member names aren’t _accurate_, even though the values are just `"1"`, `"2"`, etc... IIRC adding/changing SortText required changes elsewhere. Is it ok to use `OptionalMember` here for names that have already been included?

Reply:
symbolToSortTextMap[getSymbolId(symbol)] = SortText./*whatever you want to set*/;

Reply:
That’s what I currently have, it’s just that none of the existing SortText members describe the scenario, and I think I recall that adding a new one messed something up for @uniqueiniquity one time, but he can correct me if that’s wrong.

Reply:
Yeah I would prefer you didn't add new ones.... especially new ones above `GlobalsOrKeywords`. The other issue we had (i.e. the value for `JavaScriptIdentifiers` kept changing) is resolved now.

Reply:
I’m currently just using `OptionalMember` which is incorrect if introspecting by enum member name, but ultimately is just `"1"`.